pkg/spdk_test.go

```diff
@@ -105,7 +104,8 @@ func LaunchTestSPDKTarget(c *C, execute func(envs []string, name string, args []
 
 	if !targetReady {
 		go func() {
-			err := startTarget(GetSPDKDir(), []string{"--logflag all", "2>&1 | tee /tmp/spdk_tgt.log"}, execute)
+			logrus.Info("Starting SPDK target")
+			err := startTarget("", []string{"--logflag all", "2>&1 | tee /tmp/spdk_tgt.log"}, execute)
```

Comment:
Why is GetSPDKDir not required anymore?

Reply:
In the dapper environment, we already put spdk_tgt in the /user/loca/bin/ which is in the `PATH`, so we don't need to specify `SPDKDir` anymore.